tests/acceptance/features/apiSpaces/changingFilesShare.feature

```diff
@@ -0,0 +1,40 @@
+@api @skipOnOcV10
+Feature:
+
+  Background:
+    Given using spaces DAV path
+    And these users have been created with default attributes and without skeleton files:
+      | username |
+      | Alice    |
+      | Brian    |
+
+
+  @issue-4421
+  Scenario: Move files between shares by different users
+    Given user "Carol" has been created with default attributes and without skeleton files
+    And user "Alice" has uploaded file with content "some data" to "/textfile0.txt"
+    And user "Alice" has created folder "/PARENT"
+    And user "Brian" has created folder "/PARENT"
+    And user "Alice" has moved file "textfile0.txt" to "PARENT/shared_file.txt" in space "Personal"
+    And user "Alice" has shared folder "/PARENT" with user "Carol"
+    And user "Brian" has shared folder "/PARENT" with user "Carol"
+    And user "Carol" has accepted share "/PARENT" offered by user "Alice"
+    And user "Carol" has accepted share "/PARENT" offered by user "Brian"
+    When user "Carol" moves file "PARENT/shared_file.txt" to "PARENT (1)/shared_file.txt" in space "Shares Jail" using the WebDAV API
```

Comment:
```suggestion
    When user "Carol" moves file "PARENT/from_alice.txt" to "PARENT (1)/to_brian.txt" in space "Shares Jail" using the WebDAV API
```
Maybe something like this or anything else that makes it easier to understand that the move is happening between the folder shared by different users